src/custom-element.js

```diff
@@ -839,6 +840,20 @@ function createBaseCustomElementClass(win) {
       if (newWidth !== undefined) {
         setStyle(this, 'width', newWidth, 'px');
       }
+      if (opt_newMargins) {
+        if (opt_newMargins.top) {
```

Comment:
Should this be `opt_newMargins.top != null`? I assume `0` would be a valid value here, right?